pipelines/fuse/conf/base/catalog.yml

```diff
@@ -0,0 +1,21 @@
+# Here you can define all your data sets by using simple YAML syntax.
```

Comment:
Follow-ups:

- [ ] Configuring hardware configuration
- [ ] Using local disk (avoid writing to GCS) between tasks on same node
- [ ] Checkpointing mechanism